src/AspNetCoreModuleV2/CommonLib/GlobalVersionUtility.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+#include <experimental/filesystem>
```

Comment:
filesystem is not experimental in C++17

Reply:
There were issues with filesystem for c++17 on commandline. It kept saying std::filesystem was undefined even though all projects were targeting c++17.

Reply:
[The official microsoft docs](https://docs.microsoft.com/en-us/cpp/standard-library/filesystem) and [Community answers](https://developercommunity.visualstudio.com/content/problem/116454/c-filesystem-still-experimental-even-with-stdclate.html) say std file system is in the technical specification, but not part of the standard. I think we can still rely on it though. We use very trivial functionality of it and can adjust accordingly. cc @shirhatti